Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 
 gem 'jruby-openssl', platforms: :jruby
-gem 'rake'
+gem 'rake', '~> 10'
```

Comment:
I had a friend who used to work on the Apple QA team in the 1980s and 90s. He told me their motto was, “Quality is job 1.1.1”